src/js/components/Header/PlayingBar.react.js

```diff
@@ -158,6 +159,14 @@ export default class PlayingBar extends Component {
 
     tick() {
         this.setState({ elapsed: Player.getAudio().currentTime });
+
+        if (!Player.isThresholdReached() && Player.getAudio().currentTime >= Player.getAudio().duration * Player.getThreshold()) {
```

Comment:
If we don't do that it will increment each tick() call when `Player.getAudio().currentTime >= Player.getAudio().duration * Player.getThreshold()` so basically every 100 ms after the threshold.

Plus, this also prevent [the incrementaion if the user scroll back](https://github.com/KeitIG/museeks/issues/160#issuecomment-245395483)